karma.conf.js

```diff
@@ -53,7 +53,7 @@ module.exports = function(config) {
     // - Safari (only Mac)
     // - PhantomJS
     // - IE (only Windows)
-    browsers: ['Firefox'],
+    browsers: ['PhantomJS'],
```

Comment:
I totally agree. I think that slipped in by accident when atom decided to keep some local changes when I reverted them via git... Anyway I'll change that back asap.